frontend/httprequestworker.h

```diff
@@ -14,37 +14,39 @@
 #include <QString>
 
 // Object for request URL
-class HttpRequestInput {
+class HttpRequestInput
+{
 
-public:
-  QString url_str;
+    public:
+    QString url_str;
 
-  HttpRequestInput(QString v_url_str);
+    HttpRequestInput(QString v_url_str);
 };
 
 // Object for handling http request
-class HttpRequestWorker : public QObject {
-  Q_OBJECT
+class HttpRequestWorker : public QObject
+{
+    Q_OBJECT
 
-public:
-  QString response;
-  QNetworkReply::NetworkError error_type;
-  QString error_str;
-  QJsonDocument jsonResponse;
-  QJsonArray json_array;
+    public:
+    QString response;
+    QNetworkReply::NetworkError error_type;
+    QString error_str;
+    QJsonDocument jsonResponse;
+    QJsonArray json_array;
```

Comment:
Do these all need to be `public`, or can they be suitably hidden and made available with access functions where necessary?